### ISSUE TITLE
Prompt (InputType.decimal) doesn't allow negative numbers as input

### ISSUE THREAD

**Issue by IlSocio** [aritchie/userdialogs#263]
1) Please check all of the platforms you are having the issue on
 
 - [] iOS
 - [x] Android
 - [] UWP
 - [] PCL

2) Please state the issue to the best of your ability below
I'm using the PromptAsync method + Decimal, but the keyboard doesn't show the '-' character.
It's not possibile to insert negative numbers
(tested on LG G3)
```
                    PromptConfig config = new PromptConfig();
                    config.InputType = InputType.DecimalNumber;
                    PromptResult res = await UserDialogs.Instance.PromptAsync(config);
```

In order to allow negative numbers I had to use InputType.Name, but, of course, it shows the full-qwerty keyboard.

**Comment by MKahmen** [comment]
same for iOS.
any idea how to provide a minus sign for InputType.Decimal?

**Comment by ndesorden** [comment]
This fix is broken because 14a0c0f979817d4a2f7a867c0a5429637ecf597e

**Comment by mrbelk** [comment]
On my device's decimal keyboard, I can see a button for the "minus" sign, but it doesn't actually do anything when I tap it.
